Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN apt-get update &&\
   iptables \
   libdevmapper1.02.1 &&\
   rm -rf matching cache rm /var/lib/apt/lists/* &&\
-  pip install --upgrade --no-cache-dir wheel==0.37.1 pip==21.3.1
+  pip install --upgrade --no-cache-dir wheel==0.37.1 pip==22.0.4
```

Comment:
Can this be done as part of the start script?

Reply:
Probably, but can we just get this in as a quick fix for now, and do that later?

Reply:
moved it to entrypoint script, but waiting for ravi to test.

Reply:
Ravi tested, this is working. Lmk if you have any other change requests.